lms/djangoapps/certificates/tasks.py

```diff
@@ -4,6 +4,7 @@
 from celery_utils.logged_task import LoggedTask
 from celery_utils.persist_on_failure import PersistOnFailureTask
 from django.contrib.auth.models import User
+from lms.djangoapps.verify_student.models import SoftwareSecurePhotoVerification
```

Comment:
This seems kind of unavoidable for this PR if we decide to continue passing in a status, but just making a note: this import may complicate the effort to eventually move all of this to the openedx directory.

If we switch to passing a boolean, we no longer have that problem, but I'm guessing we probably would just be pushing the dependency further down.

Reply:
I think we'll have to have `tasks.py` continue to live in LMS in order for celery stuff to still work as intended.

Reply:
We can pass in the boolean for 'accepted' but we would still need to check the model somewhere. We could do that earlier up but that kind of confuses the logic, I think. Do you think it will make the move significantly more difficult if we kept it here?

Reply:
Thinking it through it definitely makes sense to keep tasks.py in the LMS. I'm good with keeping the logic the same as it is now, in terms of where we check models etc.